core/tests/protractor_utils/forms.js

```diff
@@ -560,19 +560,20 @@ var CodeMirrorChecker = function(elem) {
    * currentLineNumber is the current largest line number processed,
    * scrollTo is the number of pixels from the top of the text that
    * codemirror should scroll to,
+   8 Pane specifies the codemirror Left or Right Pane which is to be scrolled.
```

Comment:
The first character here should be `*`.

Reply:
Also, please follow proper capitalization within the sentence. `Left` --> `left`, `Right`--> `right`, `Pane`-->`pane`

Reply:
Done